examples/editor-sdk-document-dialect/public/dialects/american-english.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link rel="stylesheet" href="/style.css" />
+    <title>Demo App</title>
+  </head>
+  <body>
+    <h2>Document Dialect (American English)</h2>
+    <p>
+      The Grammarly Editor Plugin on this page is set to use the
+      <code>"american"</code> dialect setting.
+    </p>
+    <p>
+      <strong>Try the Grammary Editor Plugin in the text area below!</strong>
+    </p>
+    <grammarly-editor-plugin config.documentDialect="american">
+      <div contenteditable="true">
+        <p>
+          I like the color green. But not the colour blue so much.
+        </p>
+      </div>
+    </grammarly-editor-plugin>
+
+    <p>
+      To view the detected/configured dialect, click the Grammarly button, and
+      then review the dialect value in the "I write in" dropdown list. Note that
+      your users can always override this value to match their own dialect
+      preferences.
+    </p>
+    <p>
+      <a href="/index.html">Click here to return to the index.</a>
```

Comment:
```suggestion
      <a href="/index.html">Return to the index.</a>
```
Take it or leave it :-)

Reply:
Let's see what @paigedr thinks. I'm fine with either.

Reply:
I like it! Less wordy and still clear. I searched the Grammarly style guide for advice on links but did not see anything, so I think we can use our best judgment.